packages/lumx-react/src/utils/isInternetExplorer.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Check if browser is IE
+ * @return Browser is IE or not
+ */
+const isInternetExplorer = () => {
+    const ua = window.navigator.userAgent;
+    const msie = ua.indexOf('MSIE ');
+    const isIEVersion = !!navigator.userAgent.match(/Trident.*rv\:11\./);
+    return msie > 0 || isIEVersion;
+};
+
```

Comment:
Since we will be using this function as an utils, could we add a little bit of unit testing to the function?